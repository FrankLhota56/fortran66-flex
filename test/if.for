C
C IF statement tests
C
      IF (N) 98 , 99 , 100
      IF ((AND .And. Any) . or.  .Not. not) Goto 101
     0I f ( I )
     1
     2  = 1
      E     n     d
