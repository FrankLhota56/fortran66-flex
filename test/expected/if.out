Line     4: IF
Line     4: '('
Line     4: ID(n)
Line     4: ')'
Line     4: INT_LIT(98)
Line     4: ','
Line     4: INT_LIT(99)
Line     4: ','
Line     4: INT_LIT(100)
Line     5: EOS
Line     5: IF
Line     5: '('
Line     5: '('
Line     5: ID(and)
Line     5: AND
Line     5: ID(any)
Line     5: ')'
Line     5: OR
Line     5: NOT
Line     5: ID(not)
Line     5: ')'
Line     5: GO
Line     5: TO
Line     5: INT_LIT(101)
Line     6: EOS
Line     6: ID(if)
Line     6: '('
Line     6: ID(i)
Line     6: ')'
Line     8: '='
Line     8: INT_LIT(1)
Line     9: EOS
Line     9: END
